docs/integrations/open-telemetry.md

```diff
@@ -80,7 +80,6 @@ service.register(mercurius, {
       }
     }
   },
-  federationMetadata: true
```

Comment:
Comment relevant for L63: I think the schema on L63 needs changing as I believe it will error now